readme.md

```diff
@@ -54,6 +54,15 @@ of another on top of fullscreen Noita some informations:
 - [ ] Coords
 - [ ] Arbitrary entity component informations?
 
+## Development
+
+Most of the development is either done with the `nix` or `just` commands as wrapper around `cargo`:
```

Comment:
nit: Imo not clear enough that for building through nix you need to install Nix the package manager/build system, also the wording here kind of equates nix and just which are vastly different things?

eh it's good enough I'll maybe rewrite this slightly later

Reply:
Did a little rewrite :)